FantasmoSDKTests/FantasmoSDKTests.swift

```diff
@@ -267,7 +268,77 @@ class FantasmoSDKTests: XCTestCase {
         locations.append(CLLocation(latitude: 10, longitude: 10))
         XCTAssertEqual(LocationFuser.confidence(locations), .high) // 5 samples
     }
-
+    
+    func testMovementFilter() {
+        let filter = FMMovementFilter()
+        var transform = simd_float4x4(1)
+        var pixelBuffer: CVPixelBuffer? = nil
+        CVPixelBufferCreate(kCFAllocatorDefault, 64, 64, kCVPixelFormatType_OneComponent8, nil, &pixelBuffer)
+        if let nonnilBuffer = pixelBuffer {
```

Comment:
I prefer the idiom of using the same variable name in optional binding, as in https://github.com/raywenderlich/swift-style-guide#optionals

Reply:
Fixed in last commit